### ISSUE TITLE
Stops without errors

### ISSUE THREAD

**Issue by Bruteforce150** [dcmartin/hassio-addons#3]
After launch, the addon sends the configuration by MQTT, and stops, with the logs listed below(the sent configuration comes to the broker). What am I doing wrong?

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Hass.io Add-on: Motion v0.6.5
 Motion addon from LINUX package https://motion-project.github.io/
 From: DCMARTIN Hass.io add-ons for Home Assistant
 By: David C Martin <github@dcmartin.com>
-----------------------------------------------------------
 aarch64 / HassOS 2.12 / HA 0.101.2 / SU 192 / stable
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 02-updates.sh: executing... 
INFO: You are running the latest version of this add-on
[cont-init.d] 02-updates.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
NOTICE: Sat Nov 2 23:25:43 MSK 2019 /usr/bin/run.sh 
NOTICE: *** Found 1 cameras
NOTICE: Publishing configuration to 192.168.0.7 topic motion/cb7b3237-motion/start
run.sh 415 [2019-11-02T20:26:12Z] TRACE >>> mqtt_pub
run.sh 415 [2019-11-02T20:26:12Z] NOTICE >>> MQTT_DEVICE unspecified; using hostname: cb7b3237-motion.local.hass.io
run.sh 415 [2019-11-02T20:26:12Z] DEBUG >>> mosquitto_pub -i cb7b3237-motion.local.hass.io -h 192.168.0.7 -p 1883 -P MY_PASSWORD -u MY_USERNAME -r -q 2 -t motion/cb7b3237-motion/start -f /etc/motion/cb7b3237-motion.json
[cmd] /usr/bin/run.sh exited 7
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.
 
 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] syncing disks.
[s6-finish] sending all processes the TERM signal.

My config:
{
  "log_level": "info",
  "group": "motion",
  "post_pictures": "center",
  "cameras": [
    {
      "name": "hallway",
      "device": "/dev/video0",
      "url": "mjpeg://192.168.0.7/img/video.mjpeg"
    }
  ],
  "mqtt": {
    "host": "192.168.0.7",
    "port": "1883",
    "username": "MY_USERNAME",
    "password": "MY_PASSWORD"
  },
  "cloudant": {
    "url": "http://localhost:5984",
    "username": "admin",
    "password": "mysecretpassword"
  }
}

Thanks for your attention.

**Comment by dcmartin** [comment]
The current state of the addon provides support for either specifying the `url` for a network camera or an "FTP" camera; your config indicates a url for a network camera (n.b. "device" is not required); you can also add a "userpass" in the form `<user>:<password>` if your network camera requires.

IDK why you're directing the URL for the network camera back to the same host as the MQTT server; the network camera should be something that is already producing a MJPEG output; I don't support having motion process /dev/video0 on the same machine (now).

You do not need to specify the "cloudant" information at this time.

MQTT server information looks fine, presuming that 192.168.0.7 is your HA host w/ hassio starting the core-mosquitto MQTT server from the addon store.

**Comment by Bruteforce150** [comment]
Thanks for your comment. 

You are right, the ip address(192.168.0.7) belongs to the hassio host and core-mosquito MQTT server addon is installed on it.

I have a usb camera, that is connected to the hassio host. I'm trying to access it. 

Up to this time I used the motion1 addon from lovejoy777 and everything worked fine, but now I have changed the server and this addon does not work in the x64 architecture.

I'm trying to find an alternative. Perhaps I have not carefully read the description of your addon.
